doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -540,68 +511,52 @@ use one or several databases specific to your development.
 
 .. _removehints:
```

Comment:
We should introduce a new subsection title here. It doesn't make sense that these commands are listed in the "Hint databases defined in the Coq standard library" subsection.

Reply:
It made more sense to move the commands that were after the list of hint databases.  What do you think about removing the "Controlling automation" section--it's the only titled section in the chapter, all named subsections are under it.

Reply:
Eventually, we should add more sections and possibly reorganize this page a bit. But in the meantime, I'm open to anything you decide.